examples/gte/tests/test_gte_embedding.py

```diff
@@ -0,0 +1,54 @@
+from nos.client import Client
+import numpy as np
+from torch import Tensor
+
+TEST_SENTENCES = ["I'm happy", "I'm full of happiness"]
+TEST_MODEL_ID =  "TaylorAI/gte-tiny"
+
+def test_gte_embedding_embed():
+    # Create a client
+    client = Client("[::]:50051")
```

Comment:
You can setup a simple client fixture here so that you can re-use the client within this pytest: see https://github.com/autonomi-ai/nos/blob/main/examples/tutorials/02-serving-custom-methods/tests/test_model.py#L8

Reply:
Done, thanks!